src/repo.rs

```diff
@@ -77,13 +79,33 @@ pub fn repo_sync(repo: &str, force: bool) -> Result<()> {
 
     loop {
         println!("Running repo sync...");
-        let cmd = Command::new("repo")
+        let repo_sync = format!("repo sync -j{}", &num_cpus::get());
+        // `script` is a Unix command that takes a copy of all output to the terminal
+        // and writes it to `typescript` file.
+        // Below, explanation of `script` options.
+        // -q  Be quiet (do not write start and done messages to standard output).
+        // -e Return the exit status of the child process.
+        // -f Flush output after each write.
+        // -c Run the command rather than an interactive shell. This makes it easy for a
+        // script to capture the output of a program that behaves differently when its
+        // stdout is not a tty.
+        let mut cmd = Command::new("script")
             .current_dir(repo)
             .stdout(Stdio::piped())
             .stderr(Stdio::piped())
-            .args(["sync", "-j", &num_cpus::get().to_string()])
+            .args(["-qefc", &repo_sync])
             .spawn()
             .context("Failed to execute repo sync")?;
+
+        let stdout = cmd.stdout.take().unwrap();
```

Comment:
Instead of unwrap I think

```
.context("could not get stdout from script output")?; 
```
would be more appropriate

Reply:
I changed it from `unwrap()` to `context()?` .
cec8a0ba935edfb86a9237ad83a16d1d8216c6d8